src/components/footer.jsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+export function Footer({
+  user,
+  loggedIn = false,
+  onLogIn,
+  onLogOut,
+}) {
+  // TODO: Find out how to best do error handling
+  const footer = (
+    <footer className="footer">
+      <div className="container-fluid">
+        <div className="row">
+          <div className="col">
+            <p>
+              Data Hub designed and maintained by the MoTrPAC
+              BioInformatics Center at
+              <a href="https://www.stanford.edu/" target="_new"> Stanford University</a>
+            </p>
+            <p>
+              Funded by the
+              <a href="https://commonfund.nih.gov/" target="_new"> NIH Common Fund</a>
+            </p>
+          </div>
+          <div className="col copyright">
+            <p>&#169; XXXX 2018</p>
+          </div>
+          <div className="col rightAlign">
+            <AuthButton loggedInStatus={loggedIn} user={user} onLogIn={onLogIn} onLogOut={onLogOut} />
+          </div>
+        </div>
+      </div>
+    </footer>
+
+  );
+  return footer;
+}
+
+function AuthButton({
+  loggedInStatus,
+  user,
+  onLogIn,
+  onLogOut,
+}) {
+  if (loggedInStatus === true) {
```

Comment:
When evaluating a boolean, a shorthand (e.g. `if (loggedInStatus)`) can be used as well.
```suggestion
  if (loggedInStatus) {
```

Reply:
Implemented! This was initially there because of the other error you pointed out in defaultAuthProps where false was passed as a string type, meaning without the strict equality it would have passed.